Dart/src/com/jetbrains/lang/dart/validation/fixes/DartServerFixIntention.java

```diff
@@ -167,7 +169,9 @@ public boolean isAvailable(@NotNull final Project project, final Editor editor,
     final VirtualFile virtualFile = LocalFileSystem.getInstance().findFileByPath(FileUtil.toSystemIndependentName(fileEdit.getFile()));
     final ProjectFileIndex fileIndex = ProjectRootManager.getInstance(project).getFileIndex();
 
-    if (virtualFile == null || !fileIndex.isInContent(virtualFile)) return false;
+    if (fileEdit.getFileStamp() != -1) {
```

Comment:
Docs say that -1 means that a file should be created. But using nightly SDK 1.11.0-edge.161458 I see that fileEdit.getFileStamp() returns 0.